CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.3.14-M1
+### Added
+
+## fixed
+- Refactored existing models to align with AAS 3.0. Removed /query from lookup api.
```

Comment:
Add also futher information for removing /query (because it is not a part of AAS 3.0 etc).